spec/models/entry_csv_generator_spec.rb

```diff
@@ -1,18 +1,27 @@
 require "spec_helper"
+require "csv"
 
 describe EntryCSVGenerator do
-  it "generates csv" do
-    first_entry = build_stubbed(:entry)
-    second_entry = build_stubbed(:entry)
-    data = [].tap do |d|
-      d << first_entry
-      d << second_entry
-    end
+  let(:first_entry) { build_stubbed(:entry) }
+  let(:second_entry) { build_stubbed(:entry) }
+  let(:data) { [].tap do |d|
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.